README.md

```diff
@@ -113,17 +113,10 @@ $ npm install
 ### Development Mode
 We use Webpack to fire off a hot-reloading development server. This allows for continuous code changes without the need to refresh your browser.
 
-This starts the server at localhost:3000 defaulting to `NODE_ENV=development`.
+This starts the server at localhost:8080 defaulting to `NODE_ENV=development`.
 
 ```sh
-$ npm start
-```
-
-### Accessibility with React-a11y
```

Comment:
`react-a11y` is deprecated and should be replaced with `@axe-core/react` but that's not for this PR.